genrl/deep/agents/td3/td3.py

```diff
@@ -1,388 +1,396 @@
-                    for i, ep_len in enumerate(episode_len)
-                ]
-            )
-
-            self.replay_buffer.extend(zip(state, action, reward, next_state, done))
-
-            state = next_state
-
-            if np.any(done) or np.any(episode_len == self.max_ep_len):
-
-                if sum(episode) % 20 == 0:
-                    print(
-                        "Ep: {}, reward: {}, t: {}".format(
-                            sum(episode), np.mean(episode_reward), timestep
-                        )
-                    )
-
-                for i, di in enumerate(done):
-                    # print(d)
-                    if di or episode_len[i] == self.max_ep_len:
-                        episode_reward[i] = 0
-                        episode_len[i] = 0
-                        episode += 1
-
-                if self.noise is not None:
-                    self.noise.reset()
-
-                state, episode_reward, episode_len = (
-                    self.env.reset(),
-                    np.zeros(self.env.n_envs),
-                    np.zeros(self.env.n_envs),
-                )
-                episode += 1
-
-            # update params
-            if timestep >= self.start_update and timestep % self.update_interval == 0:
-                self.update_params(self.update_interval)
-
-        self.env.close()
-
-    def get_hyperparams(self) -> Dict[str, Any]:
-        hyperparams = {
-            "network_type": self.network_type,
-            "gamma": self.gamma,
-            "lr_p": self.lr_p,
-            "lr_q": self.lr_q,
-            "polyak": self.polyak,
-            "policy_frequency": self.policy_frequency,
-            "noise_std": self.noise_std,
-            "q1_weights": self.ac.qf1.state_dict(),
-            "q2_weights": self.ac.qf2.state_dict(),
-            "actor_weights": self.ac.actor.state_dict(),
-        }
-
-        return hyperparams
-
-    def load_weights(self, weights) -> None:
-        """
-        Load weights for the agent from pretrained model
-        """
-        self.ac.actor.load_state_dict(weights["actor_weights"])
-        self.ac.qf1.load_state_dict(weights["q1_weights"])
-        self.ac.qf2.load_state_dict(weights["q2_weights"])
-
-    def get_logging_params(self) -> Dict[str, Any]:
-        """
-        :returns: Logging parameters for monitoring training
-        :rtype: dict
-        """
-        logs = {
-            "policy_loss": safe_mean(self.logs["policy_loss"]),
-            "value_loss": safe_mean(self.logs["value_loss"]),
-        }
-
-        self.empty_logs()
-        return logs
-
-    def empty_logs(self):
-        """
-        Empties logs
-        """
-        self.logs = {}
-        self.logs["policy_loss"] = []
-        self.logs["value_loss"] = []
+from copy import deepcopy
+from typing import Any, Dict, Optional, Tuple, Union
+
+import gym
+import numpy as np
+import torch
+import torch.nn as nn
+
+from genrl.deep.common import (
```

Comment:
Some whitespace change here too. Not able to figure out what's new in this file. (6-7 lines are added)

Reply:
Can't see any changes too.. Its the same as the upstream
Could we just ignore it then?